src/twisted/internet/test/test_tcp.py

```diff
@@ -1230,12 +1230,26 @@ def stopReactorIfServerAccepted(_):
 
     serverConnectionMade.addCallback(stopReactorIfServerAccepted)
 
-    port = listen(reactor, serverFactory)
-    listeningHost = port.getHost()
     clientFactory = MyClientFactory()
-    connect(reactor, listeningHost, clientFactory)
 
-    reactor.callWhenRunning(exhauster.exhaust)
+    if hasattr(reactor, "callLater"):
```

Comment:
I'd probably spell this `IReactorTime.providedBy(reactor)`

Reply:
Thanks, that's cleaner!